src/landscape/landscapeSlice.js

```diff
@@ -238,6 +248,16 @@ const landscapeSlice = createSlice({
         landscape: action.payload,
         success: true,
       },
+      profile: {
+        ...state.profile,
+        landscape: {
+          ...state.profile.landscape,
+          ..._.pick(
```

Comment:
Suggestion: I think here since the `saveLandscape` could potentially be used by othere parts of the profile data, I think it should update all existing fields for the profile data not just the profileImage data, so any key that exists in the `payload`

Reply:
Actually I am not sure if this is being used, because I see the `deleteProfileImage` actions here, do we need this change in the saveLandscape action?

Reply:
> Actually I am not sure if this is being used, because I see the deleteProfileImage actions here, do we need this change in the saveLandscape action?

Yes you are right. I originally tried just using this endpoint but in the end I think it was cleaner to great a new endpoint.  I meant to remove this change but I must have overlooked it. I'll get rid of it.